src/fcu_io.cpp

```diff
@@ -302,6 +320,8 @@ void fcuIO::handle_small_imu_msg(const mavlink_message_t &msg)
                             &imu_msg.angular_velocity.z,
                             &temp_msg.temperature);
 
+  imu_msg.orientation = attitude_quat_;
```

Comment:
The reason I am using it is because of `demo_rgbd`.  demo only looks at the attitude part of the IMU message, so I thought I might as well publish it.  It's not unusual for IMU's to have internal attitude estimators (just like GPS receivers calculate heading, or the MPU6050's DMP), so I figured it wouldn't be too unusual to populate the variable.  I think we could just add a note in the documentation that the attitude portion of the message is not updated at the same rate as the IMU, and that it should be taken with a grain of salt.

I suppose an alternative would be to modify demo so that it accepts an attitude message, which makes a bit more sense, but I'm not necessarily fundamentally opposed to populating the attitude portion of the IMU message.  I mean, if we don't, then we'll just be passing around 0's.